typescript/infra/config/environments/mainnet2/validators.ts

```diff
@@ -37,18 +37,36 @@ export const validators: ChainValidatorSets<MainnetChains> = {
         },
       },
       {
-        address: '0x573b59ee4c3a20132e5a710530d1c1589290f63a',
-        name: s3BucketName('celo', 2),
+        address: '0x3bc014bafa43f93d534aed34f750997cdffcf007',
+        name: 'dsrv-celo-v2',
```

Comment:
These should be "readonly" see the v1 stanzas for an example

Reply:
Great callout